### PR TITLE
Locus Command

### PR DESCRIPTION
**Pull request by klunkerpunker**
I added the command for locus

### CODE REVIEW

File: cogs/SpecificCards.py [hunk 1]
```diff
@@ -1158,6 +1158,44 @@ async def willsSchemes(self, ctx: commands.Context):
...
+            "https://cards.scryfall.io/large/front/2/8/28603c1c-f9b4-4001-bc56-d1453d5cacf5.jpg",
+            "https://cards.scryfall.io/large/front/d/a/da785d1b-6b90-4b65-9efb-d7f329405318.jpg",
+            "https://cards.scryfall.io/large/front/c/2/c2536a4f-9e73-482b-8c1b-71974ef8950c.jpg",
+            "https://cards.scryfall.io/large/front/e/c/eca23062-6014-4a0e-8210-2e86a6308aab.jpg",
+            ("https://lh3.googleusercontent.com/d/1EsgQVM7jEAy3Yy_KvaVAUpifaEK_GtQh", "Mavren Fein, Dusk post"),
+            ("https://lh3.googleusercontent.com/d/1rcJhs0VO41VHvTY2Y80TjCQldi_3LYDN", "seachrome post"),
+            ("https://lh3.googleusercontent.com/d/1RCnsxsollhx1xmF58bt35DXVUi9NleAw", "shitpost"),
+            ("https://lh3.googleusercontent.com/d/1GFV86fBLkGZ9QPNyXGoiIenYFevXvy8p", "Omnath, Locus of the Locus")
+        ]
+        rlocus = random.randint(0, len(locusCards) - 1)
```
Comment (bones-bones) [inline cogs/SpecificCards.py:1177]: Just an fyi, python has `random.choice(my_list)` which returns a random from an array

File: cogs/SpecificCards.py [hunk 2]
```diff
@@ -1158,6 +1158,44 @@ async def willsSchemes(self, ctx: commands.Context):
...
+        #This is not super future proofed against new black border locus, if one gets printed, add it to the list under the scryfall links and add 1 to the number
+        black_border_posts = 7
+        if rlocus < black_border_posts:
+            await sendImage(locusCards[rlocus], ctx)
+        else:
+            await sendImageReply(
```
Comment (bones-bones) [inline cogs/SpecificCards.py:1183]: why send the card name?

File: cogs/SpecificCards.py [hunk 3]
```diff
@@ -1158,6 +1158,44 @@ async def willsSchemes(self, ctx: commands.Context):
...
+                url=locusCards[rlocus][0],
+                cardname=locusCards[rlocus][1],
+                message=ctx.message,
+                text=None
+            )
+
+    #And this one is for if they spell the command wrong
+    @commands.command()
```
Comment (bones-bones) [inline cogs/SpecificCards.py:1191]: absolute fire

File: cogs/SpecificCards.py (hunk 2 above)
Comment (klunkerpunker) [reply cogs/SpecificCards.py:1183]: Because right now it's not an optional argument

**Review by bones-bones — APPROVED** [review]

**Pull request merged**